pkgs/development/libraries/gssdp/default.nix

```diff
@@ -50,7 +50,8 @@ stdenv.mkDerivation rec {
     "-Dintrospection=${lib.boolToString (stdenv.buildPlatform == stdenv.hostPlatform)}"
   ];
 
-  doCheck = true;
+  # Bail out! GLib-GIO-FATAL-CRITICAL: g_inet_address_to_string: assertion 'G_IS_INET_ADDRESS (address)' failed
```

Comment:
Is this maybe related to local Networking?

Reply:
Yes, it even fails with `__darwinAllowLocalNetworking = true;`.